### PR TITLE
Changed XFCE bottom border to match other border widths (fixes #56)

### LINKED ISSUE #56: [XFCE] Thick Border Line At The Bottom Of Windows
The window border line at the bottom is considerably thicker than the borders on the left and the right.

You can fix this by resizing the bottom*.png in arc-theme/common/xfwm4/assets/

Some people say that this would impact usability, but this post explains why we should shrink the bottom border: [https://xubuntu.org/news/window-resizing-in-xubuntu-and-xfce](https://xubuntu.org/news/window-resizing-in-xubuntu-and-xfce)

### PR DESCRIPTION
**Pull request by Kirpal**
Resized bottom border in XFCE
Before:
image

After:
image

### CODE REVIEW

**Comment by NicoHood** [comment]
I remember horst originally added this to make the windows easier resizeable. I also think there were other patches that made all other borders wider, not the bottom one smaller. Have you considered that option?

**Comment by qwertychouskie** [comment]
The draggable area should be bigger than the visual border.  The border should be 1px, but the draggable area should be e.g. 8px.

**Comment by Kirpal** [comment]
I don't think xfce supports an invisible draggable area... What if all sides are 2px wide?

Also, thoughts on the topic: https://xubuntu.org/news/window-resizing-in-xubuntu-and-xfce/

**Comment by NicoHood** [comment]
If there is no invisible area we should go for 2px. Or in other words: For now we can use 2px, if someone comes up with a better idea we can improve further.

Please update the PR and post a new screenshot, so we can compare :)

**Comment by Kirpal** [comment]
Not sure I like how this ended up looking, but:
image

**Comment by NicoHood** [comment]
Other question: Does it even help for resizing, that the border is thicker?

**Comment by Kirpal** [comment]
I actually feel like it does help, especially resizing horizontally

**Comment by NicoHood** [comment]
You only changed the svg files, right? But the png files were autogenerated. I remember there was an open issue that we .gitignore all autogenerated png files. This would make the PR a lot clearer. Opinions on that? @fossfreedom 

Beside this I think we can merge this PR. horst also stated that its a good idea to increase all borders. LGTM

**Comment by Kirpal** [comment]
I like that, I was surprised that they were being tracked when I generated them. If we don't ignore them, I think I should fix it so only the pngs that need to change get changed by the commits?

**Comment by NicoHood** [comment]
Could you maybe open another PR, remove ALL pngs, add a gitignore to not track them again and test if it still builds? We could then rebase on this other PR, but I'd like to avoid to pull it into this directly (for better change tracking).

**Comment by NicoHood** [comment]
Are you able to rebase this?

**Comment by Kirpal** [comment]
I can't right now, I'll have to fix my computer at some point this weekend

File: common/xfwm4/assets-dark.svg [hunk 1]
```diff
@@ -1707,7 +1707,7 @@
     <rect
        style="opacity:1;fill:#21252b;fill-opacity:1;stroke:none;stroke-width:1;stroke-linecap:butt;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1"
        id="bottom-inactive"
-       width="2.9999716"
+       width="1"
```
Comment (NicoHood) [inline common/xfwm4/assets-dark.svg]: The comment states this is 2px, but the value is 1. Is this correct?

Comment (Kirpal) [reply common/xfwm4/assets-dark.svg]: Good catch, I don't know how that happened... Anyway, fixed now

**Comment by NicoHood** [comment]
LGTM, @fossfreedom ready to merge?

**Pull request merged**